routes/api.php

```diff
@@ -14,6 +14,27 @@
 |
 */
 
+/**
+ * Healthcheck
+ *
+ * Check that the service is up. If everything is okay, you'll get a 200 OK response.
+ *
+ * Otherwise, the request will fail with a 400 error, and a response listing the failed services.
+ *
+ * @response 400 scenario="Service is unhealthy" {"status": "down", "services": {"database": "up", "redis": "down"}}
+ * @responseField status The status of this API (`up` or `down`).
+ * @responseField services Map of each downstream service and their status (`up` or `down`).
```

Comment:
We can use the attributes version instead of the annotations for the custom endpoint configuration settings